src/mappings/entities/mapTransferManager.ts

```diff
@@ -19,25 +28,58 @@ const handleTransferManagerAdded = async (blockId: string, params: Codec[]) => {
 
   const ticker = serializeTicker(rawTicker);
   const { type, value } = getTransferManagerValue(rawManager);
+  const id = `${ticker}/${type}/${value}`;
+  const complianceType =
+    type === TransferRestrictionTypeEnum.Percentage
+      ? TransferComplianceTypeEnum.MaxInvestorOwnership
+      : TransferComplianceTypeEnum.MaxInvestorCount;
+
+  const promises = [
+    TransferManager.create({
+      id,
+      assetId: ticker,
+      type,
+      value,
+      exemptedEntities: [],
+      createdBlockId: blockId,
+      updatedBlockId: blockId,
+    }).save(),
+    TransferCompliance.create({
+      id,
+      assetId: ticker,
+      type: complianceType,
+      statTypeId: `${ticker}/${type}`,
+      value: null,
+      createdBlockId: blockId,
+      updatedBlockId: blockId,
+    }).save(),
+  ];
+
+  if (complianceType === TransferComplianceTypeEnum.MaxInvestorOwnership) {
+    promises.push(
+      StatType.create({
+        id: `${ticker}/Balance`,
+        opType: StatOpTypeEnum.Balance,
+        assetId: ticker,
+        claimType: null,
+        claimIssuerId: null,
+        createdBlockId: blockId,
+        updatedBlockId: blockId,
+      }).save()
+    );
+  }
 
-  await TransferManager.create({
-    id: `${ticker}/${type}/${value}`,
-    assetId: ticker,
-    type,
-    value,
-    exemptedEntities: [],
-    createdBlockId: blockId,
-    updatedBlockId: blockId,
-  }).save();
+  await Promise.all(promises);
 };
 
 const handleTransferManagerRemoved = async (params: Codec[]) => {
   const [, rawTicker, rawManager] = params;
 
   const ticker = serializeTicker(rawTicker);
   const { type, value } = getTransferManagerValue(rawManager);
+  const id = `${ticker}/${type}/${value}`;
 
-  await TransferManager.remove(`${ticker}/${type}/${value}`);
+  await Promise.all([TransferManager.remove(id), TransferCompliance.remove(id)]);
 };
 
 const handleExemptionsAdded = async (blockId: string, params: Codec[]): Promise<void> => {
```

Comment:
You forgot to handle the exemptions case here where we need to add to TransferComplianceExemptions